code/javascript/js-assignment-1.js

```diff
@@ -0,0 +1,15 @@
+const me = "Ngozi Clara"
+console.log("Ngozi Clara".length)
+console.log(`Hello, my name is ${me}`)
+const excitedCall = me.toUpperCase ()
+console.log(`When my friends see me they shout, ${excitedCall} `)
+const ngozi1 = "50"
+console.log(50 * 2)
+console.log(100 + 8)
+console.log(108 / 2)
+console.log(54 - 50) 
```

Comment:
It looks like you ran each command, and then manually read the output and did the next operation. We want the computer to do that work for us. Try saving the result of each operation to a variable (or doing multiple operations at the same time, using the grouping operator to preserve order of operations.) The idea is that if you wanted to use a different starting number, you could make that change on line6, and the rest of the operations would update automatically without you having to make any changes to lines 7-10.

I'd like you to try this one again please! The other two challenges on this assignment are perfect. :)